README.md

```diff
@@ -1,27 +1,26 @@
-# blogclub
-this repository is only used for our team to publish blog. if you want to be a member of the team. please see the contact section for help.
+# Blogclub
+This repository is only used for our team to publish blog. If you want to be a member of our team. Please read the contact section for help.
 
-## folder introduction
+## Folder introduction
 
-folders:
- * tech          //restore all the technical blogs   
- * english       //restore the english language practice blogs   
- * others        //restore the none technical blogs   
+Folders:
+ * Tech          //Store all the technical blogs   
+ * English       //Store the English language practice blogs   
+ * Others        //Store the none technical blogs   
  
-files:    
- 2017-08-31-the-blog-title.md    //**please follow the title partten to name your blog** 
+Files:
+ * 2017-08-31-the-blog-title.md    //**Please follow the title pattern to name your blog** 
  
-## how to contribute
+## How to contribute
 
-please follow the github pull request flow[1] to contribute your blogs.
+Please follow the github [pull request flow](https://help.github.com/articles/github-flow/) to contribute your blogs.
 
-any pull request from the team member should get reviewed by other more than two members. and all the comments from other guys must either
+Any pull request from the team member should get reviewed by more than two other members, and all the comments from other guys must either
 be fixed or get agreed by proposer. 
-we don't accept any request from anyone out of our team. thanks.
+We don't accept any request from anyone out of our team. Thanks.
 
 
-## contact
+## Contact
 
-please contact to the owner by email freesky.edward@gmail.com.
+Please contact to the owner by email freesky.edward@gmail.com.
```

Comment:
S/contact to/contact
Contact is a transitive verb. The preposition to before it is unnecessary. Consider removing it.

Reply:
done